### DISCUSSION TITLE
a donde subimos lo de "Practicas de intro a Frontend" ?

### DISCUSSION THREAD

**Discussion by RupertoTacuche** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#133] [Q&A] [answered]
Hola amigos, espero que todos esten bien. A donde subimos lo que tenemos que entregar para hoy de "practicas de intro a FrontEnd" ?

**Comment by BenitoDelaRosaChavez** [comment]
Nos van a mandar un link para poder subir cada una de ellas

**Comment by armap99** [comment]
Ya esta disponible el formulario en la sección de la semana 1 de frontend en teams.

**Comment by romarpla** [comment, marked as answer]
[Link de publicación de Teams](https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628)

Este es el link dentro del equipo de Teams para que puedan ver el link del Formulario

**Reply by RupertoTacuche** [reply]
GRacias
